src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,28 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={classNames('TodoInfo',
+        {
+          'TodoInfo--completed': completed === true,
```

Comment:
This condition would be enough. It works the same way.
```suggestion
        { 'TodoInfo--completed': completed })}
```